01-17-crafting-system/crafting/items/Item.gd

```diff
@@ -0,0 +1,6 @@
+extends Resource
+class_name Item
+
+export var item_name : String = ""
+export var recipe : Array = []
+export var crafted_item : PackedScene
```

Comment:
I think going back to JSON will make things more explicit. Be sure to name the key/property like something more explicit: `crafted_item` is a scene, it's not a crafted item. So better call it something like `Item.scene`